lab-dan/lib/fileRead.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+module.exports = {};
```

Comment:
You don't need to define module.exports as an empty object. That's the default value.